test/powershell/Language/Parser/Parsing.Tests.ps1

```diff
@@ -644,3 +644,19 @@ Describe "Parsing array that has too many dimensions" -Tag CI {
         }
     }
 }
+
+Describe "Parsing using statement with alias and linebreak" -Tag CI {
+    It "ParseError for '<Script>'" -TestCases @(
+        @{ Script = "using namespace x =`n"; ErrorId = @('MissingNamespaceAlias'); StartOffset = @(19); EndOffset = @(19) }
```

Comment:
Will this work if there are spaces after equal sign? "using namespace x =    `n"

Reply:
Yes, whitespaces are skipped over so the code only needs to explicitly handle "real" tokens.

Reply:
Nice! Please add a test with space(s) - since it is _Parser_ we should consider edge cases to avoid regressions.